subsys/fs/littlefs_fs.c

```diff
@@ -868,7 +909,7 @@ static int littlefs_unmount(struct fs_mount_t *mountp)
 
 	lfs_unmount(&fs->lfs);
 
-	if (!littlefs_on_blkdev(mountp)) {
+	if (!lfs_on_blkdev(mountp)) {
```

Comment:
Maybe the `lfs_on_fmpdev()` shall be used instead as we do have it already anyway?

Reply:
Yes, and it should also contain "IS_ENABLED". Thanks for catching this.